tests/unit/core/test_Strategy.py

```diff
@@ -0,0 +1,209 @@
+from twisted.trial import unittest
+import random
```

Comment:
Looking at the tests in here - they are more of a fuzz-test then a unit-test.  Which is fine.  If we start doing more, it might be worthwhile to make them their own test type: `tests/fuzz` in addition to `tests/unit` and `tests/functional`